serverless/messaging/reference-content/migration.mdx

```diff
@@ -152,7 +152,7 @@ See our dedicated documentation, [How to migrate to the new Messaging and Queuin
 
 SNS with Messaging and Queuing is not going into General Availability the way that NATS and SQS are. However, it has still moved from the existing old version (`v1alpha1`) to the new version (`v1beta1`) without namespaces. Therefore, even though you will not be billed for your SNS resources in the immediate future while it remains in Public Beta, there is still a migration to carry out.
 
-At the time of the release of the new version (which coincided with SQS and NATS going into General Availability) a 30 day period began. You must migrate your SNS resources from the old to the new version within this time period.
+At the time of the release of the new version (which coincided with SQS and NATS going into General Availability) a 30 day period began. You must migrate your SNS resources from the old to the new version within this time period, which ends on the 1st of December 2023.
 
 To migrate, you must **recreate your resources** in the new version. Remember that the new version of the API has no namespaces: you should simply **activate the SNS protocol** and then directly create your credentials. SQS and SNS have been separated, so if you want to use both you will need to activate both separately and create separate credentials for both. Full documentation will be available. 
```

Comment:
Maybe link to the full doc here if it is available?

Reply:
What doc do you have in mind ? 
Not sure I understand.

Reply:
I am referring to the sentence "Full documentation will be available.". I guess we could maybe change the sentence to say that it is now available (if it is , of course) and link to it.